src/_shared/utils/applyApTitleCase.ts

```diff
@@ -30,19 +44,36 @@ export const applyApTitleCase = (value: string): string => {
   if (!value) {
     return '';
   }
-  // split by separators, check if word is first or last
-  // or not blacklisted, then capitalize
-  return value
-    .split(SEPARATORS)
+
+  // Split and filter empty strings
+  // Boolean here acts as a callback, evaluates each word:
+  // If it's a non-empty string, keep the word in the array;
+  // If it's an empty string (or falsy), remove from array.
+  const allWords = value.split(SEPARATORS).filter(Boolean); // Split and filter empty strings
+
+  const result = allWords
     .map((word, index, all) => {
+      const isAfterColon = index > 0 && all[index - 1].trim() === ':';
+
+      const isAfterQuote =
+        index > 0 &&
+        (allWords[index - 1] === "'" ||
+          allWords[index - 1] === '"' ||
+          allWords[index - 1] === '\u2018' ||
+          allWords[index - 1] === '\u201C');
```

Comment:
would be good to add a comment to these unicode values as to what they represent